test/gtest/core/test_mc_reduce.h

```diff
@@ -285,30 +313,55 @@ class test_mc_reduce : public testing::Test {
 template<typename T>
 class test_mc_reduce_alpha : public test_mc_reduce<T> {};
 
-using ReductionTypesOps = ::testing::Types<
-    ReductionTest<UCC_DT_INT16, max>, ReductionTest<UCC_DT_INT32, max>,
-    ReductionTest<UCC_DT_INT64, max>, ReductionTest<UCC_DT_INT16, min>,
-    ReductionTest<UCC_DT_INT32, min>, ReductionTest<UCC_DT_INT64, min>,
-    ReductionTest<UCC_DT_INT16, sum>, ReductionTest<UCC_DT_INT32, sum>,
-    ReductionTest<UCC_DT_INT64, sum>, ReductionTest<UCC_DT_INT16, prod>,
-    ReductionTest<UCC_DT_INT32, prod>, ReductionTest<UCC_DT_INT64, prod>,
-    ReductionTest<UCC_DT_INT16, land>, ReductionTest<UCC_DT_INT32, land>,
-    ReductionTest<UCC_DT_INT64, land>, ReductionTest<UCC_DT_INT16, band>,
```

Comment:
@lappazos why we keep changing this formatting: last time you changed it this way and now  changing back.

Reply:
last time was a mistake, i fixed it here